agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -5,6 +5,16 @@
 
 from odmantic import EmbeddedModel, Field, Model, Reference
 
+class APIKeyDB(Model):
```

Comment:
The DB model should include

- The encrypted API key
- The user_id for that API key
- rate_limit
- experiation_date

Reply:
I did this earlier but thought this approach was wrong.
I reasoned that storing the user_id along with the key in the API Model would not be ideal, but instead encrypt the user_id.